src/index.js

```diff
@@ -0,0 +1,18 @@
+import form from './create-form';
+import { projectMethods, projectsArray } from './project';
+import { todoListMethods, toDoArray } from './to-do';
+const date = require('date-input-polyfill');
+import './styles.css';
+
+form.popUp();
+projectMethods.projectSubmission();
+console.log(projectsArray);
+
+projectMethods.render(projectsArray);
+projectMethods.listBtnListeners();
+
+todoListMethods.formListener();
+todoListMethods.render(toDoArray);
+// todoListMethods.deleteBtnListener();
```

Comment:
It’s better to delete the commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))